pkgs/development/python-modules/ghp-import/default.nix

```diff
@@ -0,0 +1,26 @@
+{ buildPythonPackage
+, dateutil
+, fetchPypi
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "ghp-import";
+  version = "1.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "wiqc4Qw3dT4miNFk12WnANrkuNefptsKLDEyuniBiU8=";
```

Comment:
Recent best practice I've seen has attached the SRI hash-type indicator, see e.g. r-ryantm updates. Something like ``hash = "sha256-...";

Reply:
Done.